homeassistant/components/rest/binary_sensor.py

```diff
@@ -22,6 +22,7 @@
     CONF_USERNAME,
     CONF_VALUE_TEMPLATE,
     CONF_VERIFY_SSL,
+    CONF_FILE_PATH,
```

Comment:
This name say somethings like nothing what it is. Maybe `client_certificate`?

Reply:
Yeah, had this first but i was unsure what the 'rules' regarding the extension of the const.py-File is, so i choose something that is already there. I've also updated the Docu to explain what the parameter is used for. see [https://github.com/home-assistant/home-assistant.io/pull/11238](https://github.com/home-assistant/home-assistant.io/pull/11238)

Reply:
What about it? Should I change it or not?

Reply:
The MQTT and the Roomba integration uses `certificate:`.

Reply:
I changed it to client_certificate. The certificate for Roomba+MQTT were used as truststore and not for Client authentication.

Reply:
I agree with @fabaff using `certificate` would be the best choice for consistency.